.github/workflows/update-node-js.yaml

```diff
@@ -38,16 +38,14 @@ jobs:
         run: |
           npm run update-node-js-versions
           npm run update-evergreen-config
-          git add .
-          git commit --no-allow-empty -m "chore: update node.js version" || true
 
       - name: Create Pull Request
         id: cpr
-        uses: peter-evans/create-pull-request@v6
+        uses: peter-evans/create-pull-request@5e914681df9dc83aa4e4905692ca88beb2f9e91f # 7.0.5
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          commit-message: 'chore: update node.js'
+          token: ${{ secrets.SVC_DEVTOOLSBOT_TOKEN }}
```

Comment:
This is a drive-by - I noticed the update-node-js workflow didn't trigger GHA workflow since it created a PR with the workflow token.